bundles/org.openhab.binding.etherrain/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,153 @@
+                <description>The amount of time, in minutes, that EtherRain will run zone 3 in program</description>
+                <default>0</default>
+            </parameter>
+            <parameter name="zoneOnTime4" type="integer">
+                <label>Zone 4 Run Time</label>
+                <description>The amount of time, in minutes, that EtherRain will run zone 4 in program</description>
+                <default>0</default>
+            </parameter>
+            <parameter name="zoneOnTime5" type="integer">
+                <label>Zone 5 Run Time</label>
+                <description>The amount of time, in minutes, that EtherRain will run zone 5 in program</description>
+                <default>0</default>
+            </parameter>
+            <parameter name="zoneOnTime6" type="integer">
+                <label>Zone 6 Run Time</label>
+                <description>The amount of time, in minutes, that EtherRain will run zone 6 in program</description>
+                <default>0</default>
+            </parameter>
+            <parameter name="zoneOnTime7" type="integer">
+                <label>Zone 7 Run Time</label>
+                <description>The amount of time, in minutes, that EtherRain will run zone 7 in program</description>
+                <default>0</default>
+            </parameter>
+            <parameter name="zoneOnTime8" type="integer">
+                <label>Zone 8 Run Time</label>
+                <description>The amount of time, in minutes, that EtherRain will run zone 8 in program</description>
+                <default>0</default>
+            </parameter>           
+        </config-description>
+    </thing-type>   
+    
+    <channel-type id="commandstatus">
+        <item-type>String</item-type>
+        <label>Command Status</label>
+        <description>Status of the last command given to the Etherrain</description>
+        <category>Water</category>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="operatingstatus">
+        <item-type>String</item-type>
+        <label>Operating Status</label>
+        <description>Current operating status of the Etherrain</description>
+        <category>Water</category>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="operatingresult">
+        <item-type>String</item-type>
+        <label>Operating Result</label>
+        <description>Result of operating status</description>
+        <category>Water</category>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="rainsensor">
+        <item-type>Switch</item-type>
+        <label>Rain</label>
+        <description>Provides feedback on whether the EtherRain device has detected rain or not</description>
+        <category>Sensor</category>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="runtime">
+        <item-type>Number</item-type>
```

Comment:
Please apply openHab's [Units of Measurement](https://www.openhab.org/docs/concepts/units-of-measurement.html) framework wherever applicable. This allows user interfaces greater flexibility when it comes to rendering data from your binding.
```suggestion
        <item-type>Number:Time</item-type>
```